src/http2/aws-http2-frame-settings.adb

```diff
@@ -51,7 +51,7 @@ package body AWS.HTTP2.Frame.Settings is
       Ack      : Boolean := False) return Object
    is
       Len : constant Stream_Element_Count :=
-              Stream_Element_Count (Settings'Length * Payload'Size / 8);
+              Settings'Length * Payload'Size / System.Storage_Unit;
```

Comment:
This is not correct to me. 'Size returns bits and Storage_Unit possible depends on the OS, no? The size of the object is imposed by the RFC.

Reply:
I changed it to Stream_Element'Size. It should be better as we are converting array lengths.